backend/src/main/java/com/github/liliancharlotte/controller/UserController.java

```diff
@@ -0,0 +1,32 @@
+package com.github.liliancharlotte.controller;
+
+import com.github.liliancharlotte.model.User;
+import com.github.liliancharlotte.model.UserRequest;
+import com.github.liliancharlotte.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/api")
+public class UserController {
+
+    private final UserService userService;
+
+    @Autowired
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+
+    @GetMapping("/user/{id}")
```

Comment:
Ich empfehle die Kontroller an URLs im Plural zu binden. Denn wenn man `GET /api/user` aufruft, dann bekommt man ja ein Array von Werten. Ist aber Geschmackssache: wenn Sergej und Florian das anders lehren, dann gilt ihr Wort mehr als meins ;)
```suggestion
    @GetMapping("/users/{id}")
```

Reply:
Thaaaaaaat's the way!! :raised_hands: